generate/templates/device.py.in

```diff
@@ -47,7 +47,9 @@ class PingDevice(object):
             print("Opening %s at %d bps" % (device_name, baudrate))
 
             ## Serial object for device communication
-            self.iodev = serial.Serial(device_name, baudrate)
+            # write_timeout fixes it getting stuck forever atempting to write to
+            # /dev/ttyAMA0 on Raspberry Pis, this raises an exception instead.
```

Comment:
I honestly don't know what is happening. This will fix the issue in https://github.com/bluerobotics/companion-docker/issues/321 .
device.py is getting stuck here: https://github.com/bluerobotics/ping-python/blob/master/generate/templates/device.py.in#L52

oddly it manages to open() and send_break() just fine, but attempting to write blocks forever.